src/android/io/branch/BranchSDK.java

```diff
@@ -466,12 +466,12 @@ private void createBranchUniversalObject(JSONObject options, CallbackContext cal
      * @param options A {@link JSONObject} value to set URL options.
      * @param controlParams A {@link JSONObject} value to set the URL control parameters.
      */
-    private void showShareSheet(int instanceIdx, JSONObject options, JSONObject controlParams) throws JSONException
+    private void showShareSheet(int instanceIdx, JSONObject options, JSONObject controlParams, String shareText) throws JSONException
     {
 
         Log.d(LCAT, "start showShareSheet()");
 
-        ShareSheetStyle shareSheetStyle = new ShareSheetStyle(this.activity, "Check this out!", "This stuff is awesome: ")
+        ShareSheetStyle shareSheetStyle = new ShareSheetStyle(this.activity, "Check this out!", shareText)
```

Comment:
I think we should expose the title as well instead of hard coding it to "Check this out!"